pkg/rhcos/builds.go

```diff
@@ -33,9 +33,15 @@ type metadata struct {
 }
 
 func fetchLatestMetadata(ctx context.Context, channel string) (metadata, error) {
-	build, err := fetchLatestBuild(ctx, channel)
-	if err != nil {
-		return metadata{}, errors.Wrap(err, "failed to fetch latest build")
+	var build string
+	var err error
+	if channel == DefaultChannel {
+		build = "47.165"
```

Comment:
Hmm, the PR title says 47.198, but this says 47.165. Did you mean 47.198 here?

Reply:
Oops.  I thought I'd bumped all instances after cherry picking, but obviously not.  Will fix shortly.

Reply:
Fixed with d108128 -> d54e597.